docs: fix doc store code snippet in README.md (#1389)
docarray/docarray: README.md

The Document Store interface lets you push and pull Documents to and from multiple data sources, all with the same user interface.

> As an example, let's take a look at how that would work with on-disk storage:

Reviewer: ```suggestion
For example, let's see how that works with on-disk storage:
```

The Document Store interface lets you push and pull Documents to and from multiple data sources, all with the same user interface.

As an example, let's take a look at how that would work with Jina AI Cloud storage:

```python
from docarray import BaseDoc, DocList
import os

class SimpleDoc(BaseDoc):
    text: str

# first create a personal access token here: https://cloud.jina.ai/settings/tokens

> os.environ['JINA_AUTH_TOKEN'] = 'YOUR_PAT'

Reviewer 2: will this work even if the auth token is `YOU_PAT` ?
